delta/sdk-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/views/model/IndexingData.scala

```diff
@@ -57,6 +58,32 @@ object IndexingData {
       data.copy(graph = data.graph.filter { case (s, p, _) => s == id && graphPredicates.contains(p) })
     }
 
+  def apply[A, M](
+                   exchangedValue: ConsistentWriteValue[A, M],
+                   graphPredicates: Set[Node]
+                 )(implicit cr: RemoteContextResolution, baseUri: BaseUri): IO[RdfError, IndexingData] =
+    IndexingData(exchangedValue).map { data =>
+      val id = subject(data.id)
+      data.copy(graph = data.graph.filter { case (s, p, _) => s == id && graphPredicates.contains(p) })
+    }
+
+  def apply[A, M](consistentWriteValue: ConsistentWriteValue[A, M])(implicit cr: RemoteContextResolution, baseUri: BaseUri): IO[RdfError, IndexingData] = {
+    val resource = consistentWriteValue.value.resource
+    val encoder  = consistentWriteValue.value.encoder
+    val source   = consistentWriteValue.value.source
+    val metadata = consistentWriteValue.metadata
+    val id       = resource.resolvedId
+    for {
```

Comment:
All of this is copy pasted from the other apply method. It seems like a good possibility for refactoring, given that those additions and subtractions are delicate and prone to error. We don't want to change it in one place and forget to change it in the other place